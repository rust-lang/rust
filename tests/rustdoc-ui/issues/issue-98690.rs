//@ compile-flags: --test --persist-doctests /../../ -Z unstable-options
//@ failure-status: 101
//@ only-linux

#![crate_name = "foo"]

//! ```rust
//! use foo::dummy;
//! dummy();
//! ```
