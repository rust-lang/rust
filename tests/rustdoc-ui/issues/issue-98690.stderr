Couldn't create directory for doctest executables: Permission denied (os error 13)
